### ISSUE TITLE
Blog URLs are too complex and contain unresolvable components

### ISSUE THREAD

**Issue by strogonoff** [ISO-TC211/www.isotc211.org#19]
For example, in https://www.isotc211.org/press/1997/05/14/tc211-ogc-collaborate-on-standards.html paths such as https://www.isotc211.org/press/ or https://www.isotc211.org/press/1997/ currently 404.

We probably want to **specify simpler permalinks** in the default configuration of jekyll-theme-isotc211 and make sure it gets inherited on TC sites (done via jekyll-data plugin). For existing posts on TC sites we want to **set up redirects from current URLs to new ones** (using the redirect-from plugin).

**Comment by ronaldtse** [comment]
Agreed. The current post links should be defaulted to something like `posts/2019-01-01-title`.

**Comment by ronaldtse** [comment]
Remember we want the slash at the end when generating the permalink so it generates `blah/index.html`

**Comment by strogonoff** [comment]
@ronaldtse 

> Remember we want the slash at the end when generating the permalink so it generates `blah/index.html`

My original intention is to change Jekyll’s config to specify permalinks much like on Ribose Open sites, so that canonical URLs are without file extension and with trailing slash. An URL used without trailing slash would be redirected to the canonical version.

I can’t remember for sure if RO setup involves any magic outside Jekyll (could be something with Lambda URL rewrites). Do you reckon there’s any deviation in configuration between RO and ISO TC sites? Do we use the same Terraforma configuration for both?

**Comment by ronaldtse** [comment]
Configurations are identical 😉  That's why the terraform module (https://github.com/riboseinc/terraform-aws-s3-cloudfront-website/) is so handy.

However I think in many pages we have the `permalink:` attribute set?

**Comment by strogonoff** [comment]
I’ll check on that, I believe it shouldn’t be necessary to configure permalinks in frontmatter…
